wm/desk.go

```diff
@@ -18,15 +18,34 @@ import (
 
 type x11WM struct {
 	stack
-	x      *xgbutil.XUtil
-	root   fyne.Window
-	rootID xproto.Window
-	loaded bool
+	x                   *xgbutil.XUtil
+	root                fyne.Window
+	rootID              xproto.Window
+	loaded              bool
+	moveResizing        bool
+	moveResizingStartX  int16
+	moveResizingStartY  int16
+	moveResizingGravity uint32
 
 	allowedActions []string
 	supportedHints []string
 }
 
+const (
```

Comment:
Could this be a named enum rater than uint32 - that indicates that only specific values rather than all uint32s would be allowed... (i.e. type moveResizeType uint32) 
Also is "gravity" a meaningful term here - if it can be a move instead of a resize is a more generic "type" more indicative?